### PR TITLE
CASANDRA-15663 (3.11) DESCRIBE KEYSPACE does not properly quote table names

### PR DESCRIPTION
**Pull request by adelapena**

### CODE REVIEW

File: pylib/cqlshlib/cqlhandling.py [hunk 1]
```diff
@@ -18,11 +18,58 @@
...
+    'finalfunc', 'frozen', 'function', 'functions', 'group', 'initcond', 'input', 'keys', 'keyspaces', 'language',
+    'like', 'list', 'login', 'map', 'nologin', 'nosuperuser', 'options', 'partition', 'password', 'per', 'permission',
+    'permissions', 'returns', 'role', 'roles', 'sfunc', 'static', 'storage', 'stype', 'superuser', 'trigger', 'tuple',
+    'type', 'user', 'users', 'values'
+))
+"""
+Set of unreserved keywords in CQL.
+
+Derived from .../cassandra/src/java/org/apache/cassandra/cql3/Parser.g
+"""
```
Comment (adelapena) [inline pylib/cqlshlib/cqlhandling.py]: As mentioned on the Jira ticket, I think we only need to override `cql_keywords_reserved`

File: src/antlr/Lexer.g [hunk 2]
```diff
@@ -56,7 +56,8 @@ lexer grammar Lexer;
 }
 
 // Case-insensitive keywords
-// When adding a new reserved keyword, add entry to o.a.c.cql3.ReservedKeywords as well
+// When adding a new reserved keyword, add entry to o.a.c.cql3.ReservedKeywords and
+// pylib/cqlshlib/cqlhandling.py::CqlParsingRuleSet.cql_keywords_reserved as well
```
Comment (adelapena) [inline src/antlr/Lexer.g]: Nit: I think it's `pylib/cqlshlib/cqlhandling.py::cql_keywords_reserved`

File: src/java/org/apache/cassandra/cql3/ReservedKeywords.java [hunk 3]
```diff
@@ -18,83 +18,46 @@
...
-                                                     "REPLACE",
-                                                     "DEFAULT",
-                                                     "UNSET",
-                                                     "MBEAN",
-                                                     "MBEANS"};
 
-    private static final Set<String> reservedSet = ImmutableSet.copyOf(reservedKeywords);
+    private static final String FILE_NAME = "reserved_keywords.txt";
+
+    static final Set<String> reservedKeywords = getFromResource();
```
Comment (adelapena) [inline src/java/org/apache/cassandra/cql3/ReservedKeywords.java:39]: Nit: we have missed the `@VisibleForTesting`annotation for `reservedKeywords `

File: pylib/cqlshlib/cqlhandling.py (hunk 1 above)
Comment (Gerrrr) [reply pylib/cqlshlib/cqlhandling.py]: Fixed in https://github.com/apache/cassandra/pull/1105/commits/489fe9db380d055e2b350b2c457a1fd6e0e4859b#diff-161e810fc6b29acd5307d1edbd01b314b7590c41eb92bbf6b18766588e0e0ef4L26-L58.

File: src/antlr/Lexer.g (hunk 2 above)
Comment (Gerrrr) [reply src/antlr/Lexer.g]: Fixed in https://github.com/apache/cassandra/pull/1105/commits/489fe9db380d055e2b350b2c457a1fd6e0e4859b#diff-30dc21e0dca41aab4d29072667b449dc39157d645ebba1297d0ca12ef62b713eL58-R61.

File: src/java/org/apache/cassandra/cql3/ReservedKeywords.java (hunk 3 above)
Comment (Gerrrr) [reply src/java/org/apache/cassandra/cql3/ReservedKeywords.java:39]: Fixed in https://github.com/apache/cassandra/pull/1105/commits/489fe9db380d055e2b350b2c457a1fd6e0e4859b#diff-1502b99515caea9cb7adc7f530e8fede2dcb1bd640ea322e78ed289526188bd4R38.

**Pull request closed without merging**